dist/leaflet.css

```diff
@@ -431,8 +430,7 @@ svg.leaflet-image-layer.leaflet-interactive path {
 	border-top: none;
 	line-height: 1.1;
 	padding: 2px 5px 1px;
-	font-size: 11px;
-	font-size: 0.69rem;
+	font-size: smaller;

```

Comment:
Also, do we know if relative sizes like `smaller` work in older IE versions? I'd maybe leave the adjusted pixel definitions before new ones just in case.

Reply:
Thanks for pointing this out. I thought about it but it still managed to slip my mind. I did a quick search for browser support but didn't find anything. The lack of support data is usually indicative of legacy features, but unless we find any support data we should keep the `px` variants as fallback.